pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -2611,6 +2611,9 @@ extra-packages:
   - yesod-persistent < 1.5              # pre-lts-11.x versions neeed by git-annex 6.20180227
   - yesod-static ^>= 1.5                # pre-lts-11.x versions neeed by git-annex 6.20180227
   - yesod-test ^>= 1.5                  # pre-lts-11.x versions neeed by git-annex 6.20180227
+  - immortal == 0.2.2.1                 # required by Hasura
```

Comment:
Can you annotate a hasura version here, so that we can have a ball park when to remove this? (also adding a date might make sense, but the other comments don‘t do either, so that's not a must.)